lib/xwax/timecoder.c

```diff
@@ -31,8 +31,6 @@
 #include "debug.h"
 #include "timecoder.h"
 
-#define ZERO_THRESHOLD (128 << 16)
```

Comment:
why did we drop this definition? we repeatedly use this value in the structs so it seemed useful?

Reply:
The traktor mk2 needed a different threshold from Jan's experiments. Which is why he replaced the global with the per-struct definition. Those changes to the mk2 definitions got lost during the rebase. I guess he removed the `#define` since there was no use to have multiple different global `ZERO_TRESHOLD` definitions?